setup.sh

```diff
@@ -0,0 +1,159 @@
+#!/usr/bin/env sh
+set -e
+
+sudo apt-get update
+sudo apt-get dist-upgrade --yes
+sudo apt-get install --yes \
+  asciinema \
+  atool \
+  bfs \
+  colordiff \
+  composer \
+  cowsay \
+  curl \
+  docker.io \
+  expect \
+  fonts-firacode \
+  fonts-font-awesome \
+  fonts-hack \
+  fonts-noto \
+  fonts-noto-cjk \
+  fonts-noto-extra \
+  fzf \
+  git-extras \
+  gource \
+  gnupg2 \
+  gpg \
+  htop \
+  httpie \
+  hub \
+  jq \
+  make \
+  net-tools \
+  nodejs \
+  npm \
+  numlockx \
+  papirus-icon-theme \
+  php \
+  polybar \
+  ripgrep \
+  rofi \
+  ruby \
+  ruby-bundler \
+  screenfetch \
+  ssh-audit \
+  steam \
+  subversion \
+  tmux \
+  vim \
+  wget \
+  xfce4-screenshooter \
+  zsh
+
+# Google Chrome
+sudo sh -c 'echo "deb [arch=amd64] https://dl.google.com/linux/chrome/deb/ stable main" > /etc/apt/sources.list.d/google-chrome.list'
+wget -O- https://dl-ssl.google.com/linux/linux_signing_key.pub | sudo tee /etc/apt/trusted.gpg.d/linux_signing_key.pub
+sudo apt-key adv --keyserver keyserver.ubuntu.com --recv-keys 78BD65473CB3BD13
+sudo apt-get update
+sudo apt-get install google-chrome-stable
+
+# Snap only packages.
+# And yes, we have to do this shitty commands chaining because
+# snap does not understand having an already installed package
+# is not that a deal.
+snap list | awk '{print $1}' | grep -q "^discord$" || snap install discord
+snap list | awk '{print $1}' | grep -q "^slack$" || snap install slack
+snap list | awk '{print $1}' | grep -q "^code$" || snap install --classic code
+
+# Regolith desktop
+# https://regolith-desktop.com/#get-regolith-22
+wget -qO - https://regolith-desktop.org/regolith.key | \
+gpg --dearmor | sudo tee /usr/share/keyrings/regolith-archive-keyring.gpg > /dev/null
+echo deb "[arch=amd64 signed-by=/usr/share/keyrings/regolith-archive-keyring.gpg] \
+https://regolith-desktop.org/release-ubuntu-jammy-amd64 jammy main" | \
+sudo tee /etc/apt/sources.list.d/regolith.list
+sudo apt-get update
+sudo apt-get install regolith-desktop regolith-compositor-picom-glx
```

Comment:
# TODO: Graphics glitches

We may have some graphic glitches by moving the pointer around some UI interface like vscode folder list or context menu elements.

WE may have to try different compositors: https://regolith-desktop.com/docs/using-regolith/configuration/#compositors

Reply:
Resolved just by removing the currently installed compositor. I didn't try the other. It may works, but I don't really need more fancy animation.